assets/src/apps/authoring/store/activities/actions/saveActivity.ts

```diff
@@ -23,80 +23,86 @@ import { diff } from 'deep-object-diff';
 export const saveActivity = createAsyncThunk(
   `${ActivitiesSlice}/saveActivity`,
   async (payload: { activity: IActivity; undoable?: boolean }, { dispatch, getState }) => {
-    const { activity, undoable = true } = payload;
-    const rootState = getState() as any;
-    const projectSlug = selectProjectSlug(rootState);
-    const resourceId = selectResourceId(rootState);
-    const currentActivityState = selectActivityById(rootState, activity.id) as IActivity;
-    const group = selectCurrentGroup(rootState);
+    try {
```

Comment:
Errors in here were getting swallowed with no indication there was an error, now at least a console message goes out.